app/webpacker/components/Panel/pages/BannedCompetitorsPage/BannedCompetitors.jsx

```diff
@@ -20,6 +20,8 @@ export default function BannedCompetitors({
             <Table.HeaderCell width={5}>User</Table.HeaderCell>
             <Table.HeaderCell width={2}>Start date</Table.HeaderCell>
             <Table.HeaderCell width={2}>End date</Table.HeaderCell>
+            <Table.HeaderCell width={2}>Ban reason</Table.HeaderCell>
+            <Table.HeaderCell width={2}>Ban scope</Table.HeaderCell>
```

Comment:
For consistency with the other header cells, eg "Start date" means "Ban start date" already:
```suggestion
            <Table.HeaderCell width={2}>Start date</Table.HeaderCell>
            <Table.HeaderCell width={2}>End date</Table.HeaderCell>
            <Table.HeaderCell width={2}>Reason</Table.HeaderCell>
            <Table.HeaderCell width={2}>Scope</Table.HeaderCell>
```